function/pkg/operator/triggers_operator.go

```diff
@@ -30,7 +58,10 @@ func (t triggersOperator) Apply(ctx context.Context, opts ApplyOptions) error {
 	if !found {
```

Comment:
Do we still need the `ownerID` label? Couldn't it be removed now that we're no longer using it for deletion?